### PR TITLE
Add Guide to Emulator Setup for Developers

### PR DESCRIPTION
**Pull request by tomojira**
Added a guide on how to set up RAIntegration with BizHawk, DuckStation, PCSX2, PPSSPP, and Dolphin.

### CODE REVIEW

File: docs/.vitepress/config/en.ts [hunk 1]
```diff
@@ -255,6 +255,10 @@ function sidebarDeveloperDocs(): DefaultTheme.SidebarItem[] {
           text: "Junior Developer Program",
           link: "/developer-docs/jr-dev-rules",
         },
+        {
+          text: "Emulator Setup for Developers",
+          link: "/developer-docs/emulator_setup_for_developers",
```
Comment (wescopeland) [inline docs/.vitepress/config/en.ts]: I think this needs to be separated by dashes, otherwise it produces a broken link.

**Comment by wescopeland** [comment]
Good idea with the screenshots. Unfortunately I think something may have gone wrong when uploading them 🤔 This is what I'm seeing:

Screenshot 2024-06-17 at 9 13 42 PM

In the PR itself (check the "Files changed" tab), it looks like the issue surfaces as well:

Screenshot 2024-06-17 at 9 14 01 PM

**Comment by tomojira** [comment]
Oops, sorry. Not sure what happened with the images but I think they work now after uploading them again.

**Review by wescopeland — APPROVED** [review]
No worries, that is what peer review is for :-)
Looks good to me!

**Pull request merged**